explore-analyze/report-and-share/automating-report-generation.md

```diff
@@ -159,3 +165,64 @@ If you experience issues with the deprecated report URLs after you upgrade {{kib
 :::{important}
 In earlier {{kib}} versions, you could use the `&sync` parameter to append to report URLs that held the request open until the document was fully generated. The `&sync` parameter is now unsupported. If you use the `&sync` parameter in Watcher, you must update the parameter.
 :::
+
+## Schedule and share reports [schedule-report-generation]
+
+```{applies_to}
+stack: preview 9.1
```

Comment:
Should there be a serverless `applies_to` tag? I noticed Serverless is mentioned in the requirements.

Reply:
It's not mandatory as per the guidelines since it's implied from the page's frontmatter that all of the content applies to serverless unless specifically called out, but up to you @nastasha-solomon. I think it doesn't hurt to also add it just to avoid users think this is for stack 9.1 ONLY.

Reply:
Thanks, both! I figured that adding Serverless would be redundant since it's already mentioned in the front matter. But I can also see how users might miss that if they land directly on this section, or are still learning how the `applies to` tags work.

```suggestion
stack: preview 9.1
serverless: preview
```

Reply:
Indeed if the feature described specifically in this section is in preview state in serverless, then it needs to be specifically called out 👍

Reply:
Ah, good point. Thanks for the reminder.